From: Will Thompson <will@willthompson.co.uk>
Date: Fri, 29 Nov 2019 20:40:14 +0000
Subject: [PATCH] desktop: add X-Endless-LaunchMaximized=false

Endless OS launches applications maximized by default under certain
conditions.

Gnote looks ridiculous maximized. Let's not do this.

--- a/data/org.gnome.Gnote.desktop.in.in	2021-03-29 11:43:48.979429328 +1100
+++ b/data/org.gnome.Gnote.desktop.in.in	2021-03-29 11:44:15.248495328 +1100
@@ -15,3 +15,4 @@
 X-GNOME-Bugzilla-Product=gnote
 X-GNOME-Bugzilla-Component=main
 X-GNOME-Bugzilla-Version=@GNOTE_VERSION@
+X-Endless-LaunchMaximized=false
